### PR TITLE
Feature/3DS 2

### PR DESCRIPTION
**Pull request by mreams-bambora**
Production release of 3DS 2 docs!

### CODE REVIEW

File: source/includes/na/take_payments/_payment_3d_secure.md [hunk 1]
```diff
@@ -5,20 +5,37 @@ Definition
...
+         "browser": {
+            "accept_header": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3",
+            "java_enabled": "false",
+            "language": "en-US",
+            "color_depth": "24bits",
+            "screen_height": 1080,
+            "screen_width": 1920,
+            "time_zone": -120,
+            "user_agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.90 Safari/537.36",
+            "Javascript_enabled": true,
```
Comment (Bambora-Dave-Robinson) [inline source/includes/na/take_payments/_payment_3d_secure.md]: This needs to begin with a lowercase 'j'

File: source/includes/na/take_payments/_payment_3d_secure.md [hunk 2]
```diff
@@ -5,20 +5,37 @@ Definition
 POST /v1/payments HTTP/1.1
 
 Request
-curl https://api.na.bambora.com/v1/payments
--H "Authorization: Passcode MzAwMjAwNTc4OjRCYU..."
--H "Content-Type: application/json"
+curl https://api.na.bambora.com/v1/payments \
+-H "Authorization: Passcode MzAwMjAwNTc4OjRCYUQ4MkQ5MTk3YjRjYzRiNzBhMjIxOTExZUU5Zjcw" \
```
Comment (Bambora-Dave-Robinson) [inline source/includes/na/take_payments/_payment_3d_secure.md:9]: Is there a reason we've expanded this to a full passcode as opposed to one suffixed with the '...' as before? Probably doesn't matter but might be better to keep it consistent with what we had before, unless it was changed for a specific reason

File: data/na/swagger/1-0-5.yaml [hunk 3]
```diff
@@ -745,6 +745,8 @@ definitions:
         description: Security code on the back of the credit card. This can be set to mandatory in the back office.
       3d_secure:
          $ref: '#/definitions/3DSecure'
+      broswer_data:
```
Comment (Bambora-Dave-Robinson) [inline data/na/swagger/1-0-5.yaml]: Typo

File: source/includes/na/take_payments/_payment_3d_secure.md (hunk 2 above)
Comment (mreams-bambora) [reply source/includes/na/take_payments/_payment_3d_secure.md:9]: No specific reason, I'll add that to the list of non-urgent fixes :)

**Review by Jim-Edwards-Worldline — APPROVED** [review]

**Pull request merged**